docs/content/docs/dev/table/concepts/versioned_tables.md

```diff
@@ -152,9 +153,9 @@ WHERE rownum = 1;
 +(INSERT)        09:00:00      Yen        102
 +(INSERT)        09:00:00      Euro       114
 +(INSERT)        09:00:00      USD        1
-+(UPDATE_AFTER)  10:45:00      Euro       116
 +(UPDATE_AFTER)  11:15:00      Euro       119
-+(INSERT)        11:49:00      Pounds     108
++(INSERT)        11:45:00      Pounds     107
++(UPDATE_AFTER)  11:49:00      Pounds     108
```

Comment:
Why we need change here?

Reply:
it's better to add more 'update_after' lines (not just one) for better understanding